packages/renderless/src/recycle-scroller/index.ts

```
@@ -414,32 +420,32 @@ const computeRange = (args) => {
+      lower = cursor
+    } else if (cursor < count - 1 && sizes[cursor + 1].accumulator > scroll.start) {
+      upper = cursor
     }
 
-    i = ~~((a + b) / 2)
-  } while (i !== oldI)
+    cursor = ~~((lower + upper) / 2)
+  } while (cursor !== oldCursor)
 
-  i < 0 && (i = 0)
+  cursor < 0 && (cursor = 0)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)